lib/expand/fsmspec.go

```diff
@@ -92,6 +93,15 @@ func FSMSpec(config FSMConfig) fsm.FSMSpecFunc {
 				config.Operator,
 				config.Runner)
 
+		case p.Phase.ID == installphases.ExportPhase:
```

Comment:
All the other cases check if the phase prefixes the ID. Is it okay to compare for an exact match here?

Reply:
It is - as opposed to installation case, the export step during expand is for the local node only, hence no node suffix is used (it's just `/export`). But looking at the surrounding code I guess you're right to keep that consistent.